### PR TITLE
CI: Use misspell binary

### PR DESCRIPTION
**Pull request by haya14busa**
Building misspell might use lots of resource and be killed.
https://github.com/vim-jp/vital.vim/pull/513#issuecomment-312435191

ref: https://github.com/client9/misspell#install

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ build:
       - REVIEWDOG_GITHUB_API_TOKEN=$$REVIEWDOG_GITHUB_API_TOKEN
     commands:
       - go get -d -v ./scripts
-      - go get -u github.com/client9/misspell/cmd/misspell
+      - curl -L https://git.io/misspell | sh
```
Comment (rhysd) [inline .drone.yml]: From [misspell's installation document](https://github.com/client9/misspell#install),

> Both will install as ./bin/misspell.

So we might need to specify `misspell` executable with relative path in `reviewdog.yml`.

File: autoload/vital.vim [hunk 2]
```diff
@@ -10,3 +10,5 @@ function! vital#of(name) abort
   endif
   return vital#_{substitute(ver[0], '\W', '', 'g')}#new()
 endfunction
+
+let g:exsiting_variable = 1
```
Comment (haya14busa) [inline autoload/vital.vim]: :bird: :+1:

File: .drone.yml (hunk 1 above)
Comment (haya14busa) [inline .drone.yml]: Oops, thank you for the catch!

**Comment by haya14busa** [comment]
Fixed. Please take another look.

**Pull request merged**